sirepo/package_data/static/js/sirepo.js

```diff
@@ -96,6 +99,58 @@ angular.element(document).ready(function() {
             : addTag(src, 'script', 'body', 'src', {'type': 'text/javascript', 'async': true});
     }
 
+    function loadFromPath(path, type, callback, errCallback, finallyCallback) {
+        let d = $.Deferred();
+        $.get(`${path}${SIREPO.SOURCE_CACHE_KEY}`, function (res) {
+            callback(res);
+        }, type)
+            .fail(function (res) {
+                if (errCallback) {
+                    errCallback(res);
+                }
+            })
+            .always(function (res) {
+                if (finallyCallback) {
+                    finallyCallback(res);
+                }
+                d.resolve();
+            });
+        return d.promise();
+    }
+
+    function loadBlock(b) {
+        let val = null;
+        let status = 500;
```

Comment:
seems like `status` should just be the name `ok` and set to false unless it is set to true below.

Reply:
60e7ccc